sqlitecluster/SQLite.h

```diff
@@ -23,7 +23,7 @@ class SQLite {
     // maxRequiredJournalTableID: This is the maximum journal table ID that we'll verify. If it's -1, we'll only verify
     //                            'journal' and no numbered tables.
     SQLite(const string& filename, int cacheSize, int autoCheckpoint, int maxJournalSize, int journalTable,
-           int maxRequiredJournalTableID);
+           int maxRequiredJournalTableID, const string& synchronous);
```

Comment:
Give this a default value of `""`, then you don't need to explicitly pass it everywhere (like in the tests).